vendor/assets/manifest.webapp

```diff
@@ -32,6 +32,18 @@
       "folder": "/",
       "index": "index.html",
       "public": false
+    },
+    "/": {
```

Comment:
We cannot duplicate the route `/`, as it is already defined above. I suggest using something like `/service-create` or `/#service-create`.

The first one `/service-create` implies that we create a second access file to my-accounts and add some webpack config to generate it (see https://github.com/gregorylegarec/cozy-intent-example/blob/master/config/webpack.config.base.js#L56).

The second option `/#service-create` implies that we just create a new route in our app. I think it is the simpler solution for now, but I never tested it and I am not sure that it could work (ping @nono)

I'd love to have @m4dz opinion about those two options, as we already talked about during grooming but I don't remember having made a decision.